src/ModelContextProtocol.Core/AIContentExtensions.cs

```diff
@@ -365,12 +371,15 @@ public static IList<AIContent> ToAIContents(this IEnumerable<ResourceContents> c
 
     /// <summary>Creates a new <see cref="ContentBlock"/> from the content of an <see cref="AIContent"/>.</summary>
     /// <param name="content">The <see cref="AIContent"/> to convert.</param>
+    /// <param name="options">The <see cref="JsonSerializerOptions"/> to use for serialization. If <see langword="null"/>, <see cref="McpJsonUtilities.DefaultOptions"/> is used.</param>
     /// <returns>The created <see cref="ContentBlock"/>.</returns>
     /// <exception cref="ArgumentNullException"><paramref name="content"/> is <see langword="null"/>.</exception>
-    public static ContentBlock ToContentBlock(this AIContent content)
+    public static ContentBlock ToContentBlock(this AIContent content, JsonSerializerOptions? options = null)
```

Comment:
@stephentoub how concerned are we about this kind of breaking change? Should we change this to be a new overload?